corehq/apps/app_manager/tests/test_tasks.py

```diff
@@ -0,0 +1,143 @@
+import json
+import os
+
+from django.test import TestCase
+from unittest.mock import patch
+
+from corehq.apps.data_dictionary.models import CaseProperty, CaseType
+from corehq.apps.app_manager.dbaccessors import get_app, get_build_ids
+from corehq.apps.app_manager.models import ConditionalCaseUpdate, import_app
+from corehq.apps.app_manager.tasks import (
+    autogenerate_build,
+    prune_auto_generated_builds,
+    refresh_data_dictionary_from_app,
+)
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+from corehq.apps.app_manager.tests.util import add_build, get_simple_form, patch_validate_xform
+from corehq.apps.app_manager.views.releases import make_app_build
+from corehq.apps.domain.shortcuts import create_domain
+
+
+@patch_validate_xform()
+class AppManagerTasksTest(TestCase):
+    file_path = ('data',)
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.build1 = {'version': '1.2.0', 'build_number': 7106}
+        cls.build2 = {'version': '2.7.0', 'build_number': 20655}
+
+        add_build(**cls.build1)
+        add_build(**cls.build2)
```

Comment:
I believe these also save couch objects. Sorry I didn't make that mare clear in my previous comment.

Reply:
It turns out these weren't used. I had ported them over because I got hem confused with local variables also named `build1` and `build2`.